test/unit/test_type.c

```diff
@@ -0,0 +1,546 @@
+// SPDX-FileCopyrightText: 2018 ret2libc <sirmy15@gmail.com>
```

Comment:
what did i do in this file? mmm

Reply:
It was extract from `test/unit/test_util.c` which had you in copyrights.